docs/Modelagem/ModelagemDinamica/2.2.1.DiagramaSequencia.md

```diff
@@ -0,0 +1,96 @@
+| **Especificação de Execução** | Representa o intervalo de tempo em que uma entidade executa uma operação ou está envolvida em uma atividade, como executar um comportamento ou ação dentro da linha de vida, enviar um sinal para outro participante, ou aguardar uma mensagem de resposta de outro participante. É indicada como um retângulo alongado sobre a linha de vida. | ![exemplo-especificao](assets/exemplo-especificacao.png) |
+| **Fragmento Combinado** | Define uma combinação (expressão) de fragmentos de interação, um fragmento combinado é definido por um operador de interação e operandos de interação correspondentes, como o Loop (utilizado para ações que se repetem), Opt (usado para ações opcionais que podem ou não ocorrer, dependendo de uma condição), Alt (utilizado em cenários alternativos, onde apenas uma opção será seguida). | ![exemplo-fragmento](assets/exemplo-fragmento.png) |
+| **Ocorrência de Destruição** | Representa o momento em que uma entidade é destruída ou deixa de existir na interação, e é indicada por um "X" no final da linha de vida da entidade. | ![exemplo-destruicao](assets/exemplo-destruicao.png) |
+
+## Diagramas
+
+### Solicitação de Pedido
+
+<center>
+
+![diagrama-sequencia-pedido](assets/sequencia_pedido.png)
+
+**Autores**: [Felipe Amorim de Araújo](https://github.com/lipeaaraujo), [Lucas Martins Gabriel](https://github.com/martinsglucas), [Guilherme Silva Dutra](https://github.com/GuiDutra21)
+</center>
+
+### Cadastro de Usuário
+
+<center>
+
+![diagrama-sequencia-cadastro](assets/sequencia-cadastro.png)
+
+**Autores**: [Felipe Amorim de Araújo](https://github.com/lipeaaraujo), [Lucas Martins Gabriel](https://github.com/martinsglucas)
+
+</center>
+
+### Login
+
+![diagrama-sequencia-login](assets/sequencia-login.png)
+
+<center>
+
+**Autores**: [Felipe Amorim de Araújo](https://github.com/lipeaaraujo)
+
+</center>
+
+### Logout
+
+![diagrama-sequencia-logout](assets/sequencia-logout.png)
+
+<center>
+
+**Autores**: [Felipe Amorim de Araújo](https://github.com/lipeaaraujo)
+
+</center>
+
+### Editar conta
+
+![diagrama-sequencia-conta](assets/sequencia-conta.png)
+
+<center>
+
+**Autores**: [Davi Gonçalves Akegawa Pierre](https://github.com/DaviPierre), [Raquel Ferreira Andrade](https://github.com/raquel-andrade)
+
+</center>
+
+### Cadastro método de pagamento
+
+![diagrama-sequencia-pagamento](assets/sequencia-pagamento.png)
+
+<center>
+
+**Autores**: [Davi Gonçalves Akegawa Pierre](https://github.com/DaviPierre), [Raquel Ferreira Andrade](https://github.com/raquel-andrade)
+
+</center>
+
```

Comment:
## Revisão 

<iframe width="560" height="315" src="https://www.youtube.com/embed/l6QLOzLHXv8?si=DAx0Jc0tBi-1Djb8" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>